packages/core/src/utils/requestQueue.test.ts

```diff
@@ -119,6 +119,7 @@ test("onIdle() resolves once idle", async ({ networks, common }) => {
   queue.request({ method: "eth_chainId" });
   queue.request({ method: "eth_chainId" });
 
+  queue.clear();
```

Comment:
Why did you need to add these?

Reply:
More accurately reflects how it is used in the actual codebase